src/core/reducers/addons.js

```diff
@@ -22,8 +54,12 @@ export function denormalizeAddon(apiAddon) {
   return apiAddon;
 }
 
-export default function addon(state = initialState, action) {
+export default function addon(
+  state: Object = initialState,
+  action: Object = {}
```

Comment:
heh, yeah. I will adjust this when fixing the reducer. I'll also move `loadEntities` over here but only after bathing it in bleach and renaming it to `loadAddons` 💥